test/node/src/clear.test.ts

```diff
@@ -0,0 +1,192 @@
+import { BUILT_IN_DIALECTS, initTest, TestContext, testSql } from './test-setup'
```

Comment:
Run prettier for this file

Reply:
We should add a linter

Reply:
`npm run prettier` does not format this file. I don't know why, but I do it manually anyway.